include/fg/chart.h

```diff
@@ -0,0 +1,191 @@
+/*******************************************************
+ * Copyright (c) 2015-2019, ArrayFire
+ * All rights reserved.
+ *
+ * This file is distributed under 3-clause BSD license.
+ * The complete license agreement can be obtained at:
+ * http://arrayfire.com/licenses/BSD-3-Clause
+ ********************************************************/
+
+#pragma once
+
+#include <fg/defines.h>
+#include <fg/image.h>
+#include <fg/plot.h>
+#include <fg/surface.h>
+#include <fg/histogram.h>
+
+#include <string>
+#include <vector>
+
+namespace internal
```

Comment:
This is a bad name for a name space. it should either be nested inside `forge` or just use `void *` instead of _Chart.

Also considering that it is inside a different namespace, why the `_` ?